Library/Homebrew/global.rb

```diff
@@ -30,6 +31,9 @@
 module Homebrew
   extend FileUtils
 
+  @messages = Messages.new
+  class << self; attr_accessor :messages; end
```

Comment:
There's a `class << self` just below here it might be nicer for all this to live in (and perhaps a lazy initialisation of `@messages`

Reply:
Amended to address.